test/unit_test/CMakeLists.txt

```diff
@@ -34,7 +35,6 @@ add_executable(vw-unit-test.out
   chain_hashing.cc
   offset_tree_tests.cc
   random_test.cc
-  cats_tree_tests.cc
```

Comment:
Was this intentional?

Reply:
Yes, cats_tree_test.cc is also added at the top of the file